sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -14,6 +14,16 @@
 
 namespace Azure { namespace Core { namespace Http {
 
+  namespace _detail {
+    /**
+     * @brief Default maximum time in milliseconds that you allow the connection phase to the server
+     * to take.
+     *
+     */
+    AZ_CORE_DLLEXPORT static const std::chrono::milliseconds DefaultConnectionTimeout
+        = std::chrono::milliseconds(300000);
```

Comment:
You can use much more readable `= std::chrono::minutes(5)` instead of `milliseconds(300000)`.

Reply:
```suggestion
        = std::chrono::minutes(5);
```